src/lib/Liblog/pbs_log.c

```diff
@@ -722,17 +723,18 @@ log_err(int errnum, const char *routine, const char *text)
 
 #ifdef WIN32
 		LPVOID	lpMsgBuf;
-		int	err = GetLastError();
+		DWORD	err = GetLastError();
 		int		len;
 
+		snprintf(buf, LOG_BUF_SIZE, "Err(%lu): ", err);
```

Comment:
I have one suggestion. Can you add this printing of error code in log_joberr function too ? https://github.com/PBSPro/pbspro/blob/1324a6c8842903b9412ecd6dfb618e7f3a725cbf/src/lib/Liblog/pbs_log.c#L842
Over there also we are calling this FormatMessage function but not storing the error code.

Reply:
Hey @sakshamgarg Thanks for review. Sure. I understand that may help you while refactoring resmom and resmom_win. Please review the latest commit and I changed the PR scope also.